openpype/hosts/houdini/api/lib.py

```diff
@@ -836,6 +836,70 @@ def get_current_context_template_data_with_asset_data():
     return template_data
 
 
+def get_houdini_color_settings():
+    """Get Houdini working file color settings.
+
+    Returns:
+        Dict: The dictionary contains the Houdini working file color settings
+              if the settings are enabled, otherwise it is an empty dictionary.
+    """
+
+    project_settings = get_current_project_settings()
+    color_settings = project_settings["houdini"]["imageio"]["workfile"]
+    if color_settings["enabled"]:
+        color_settings.pop("enabled")
+        # Remove leading, and trailing whitespaces
+        view_Space = color_settings["review_color_space"]
+        color_settings["review_color_space"] = view_Space.strip()
+        return color_settings
+
+    return {}
+
+
+def set_review_color_space(opengl_node, log=None):
+    """Set ociocolorspace parameter for the given OpenGL node.
+
+    if workfile settings are enabled, it will use the value
+    exposed in the settings.
+
+    if workfile settings are disabled, it will use the default
+    colorspace corresponding to the display & view of
+    the current Houdini session.
+
+    Args:
+        OpenGl node (hou.Node): ROP node to set its ociocolorspace parameter.
+        log (logging.Logger): Logger to log to.
+    """
+
+    if log is None:
+        log = self.log
+
+    # Set Color Correction parameter to OpenColorIO
+    if opengl_node.evalParm("colorcorrect") != 2:
+        opengl_node.setParms({"colorcorrect": 2})
```

Comment:
```suggestion
        opengl_node.parm("colorcorrect").set(2)
```

Reply:
I'd probably even do:
```python
colorcorrect_parm = opengl_node.parm("colorcorrect")
if colorcorrect_parm.eval() != 2:
   color_correct.parm.set(2)
```
Although I probably wouldn't even do the check if I knew I always wanted it to be "2" regardless